storage/buckets.rb

```diff
@@ -90,6 +90,86 @@ def create_bucket project_id:, bucket_name:
   # [END create_bucket]
 end
 
+def create_bucket_class_location project_id:, bucket_name:, location:, storage_class:
+  # [START create_bucket_class_location]
+  # project_id    = "Your Google Cloud project ID"
+  # bucket_name   = "Name of Google Cloud Storage bucket to create"
+  # location      = "Location of your Google Cloud Storage bucket"
+  # storage_class = "Storage class of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.create_bucket bucket_name,
+                                  location:      location,
+                                  storage_class: storage_class
+
+  puts "Created bucket #{bucket.name} in #{location} with storage class #{storage_class}"
+  # [END create_bucket_class_location]
+end
+
+def get_bucket_labels project_id:, bucket_name:
+  # [START get_bucket_labels]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.bucket bucket_name
+
+  puts "Updated labels for #{bucket_name}"
+  bucket.labels.each do |key, value|
+    puts "#{key} = #{value}"
+  end
+  # [END get_bucket_labels]
+end
+
+def add_bucket_label project_id:, bucket_name:, label_key:, label_value:
+  # [START add_bucket_label]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+  # label_key   = "Cloud Storage bucket Label Key"
+  # label_value = "Cloud Storage bucket Label Value"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.bucket bucket_name
+
+  bucket.update do |bucket|
+    bucket.labels[label_key] = label_value
+  end
+
+  puts "Updated labels for #{bucket_name}"
```

Comment:
Why list the labels here? The sample demonstrates *adding*

Can we just `puts "Added label to bucket"?